index.html

```diff
@@ -996,8 +996,8 @@ <h2>
         creation of the {{ArrayBuffer}} object are re-thrown.
       </p>
       <p data-cite="encoding">
-        The <dfn>json()</dfn> method steps are to return the result of <a>parsing JSON bytes to a
-        JavaScript value</a> given [=this=]'s [=bytes=].
+        The <dfn>json()</dfn> method steps are to return the result of [=parse JSON bytes to a
+        JavaScript value|parsing JSON bytes to a JavaScript value=] given [=this=]'s [=bytes=].
```

Comment:
While approving, I thought this should just work on ReSpec too? cc @sidvishnoi

Reply:
It's relatively new that it works in Bikeshed. As this doesn't really block the other PRs in terms of preview or reviewability, I can wait a bit to see what @sidvishnoi says.

Reply:
This is based on [`textVariations`](https://github.com/speced/respec-web-services/blob/52c8d47f9545c9a25015f2fead6f3121abc10595/routes/xref/lib/utils.ts#L8) (based on Bikeshed itself, but maybe needs some updates now). It can only do these transformations at the end of string (if it was "do the parse", it could have become "do the parsing").
Not planning to support transformations at the start any time soon though.